vpplink/types/cnat.go

```diff
@@ -34,6 +35,18 @@ const (
 	MaglevLB  = CnatLbType(cnat.CNAT_LB_TYPE_MAGLEV)
 )
 
+type CnatHashConfig uint8
```

Comment:
Maybe worth adding a `String()` function on the `CnatHashConfig` type

Reply:
Right, there is this, but the output will probably be really ugly...
e.g. usually `"IP_API_FLOW_HASH_SRC_IP|IP_API_FLOW_HASH_DST_IP|IP_API_FLOW_HASH_SRC_PORT|IP_API_FLOW_HASH_DST_PORT|IP_API_FLOW_HASH_PROTO"` ...